tck/features/clauses/call/Call5.feature

```diff
@@ -29,3 +29,102 @@
 #encoding: utf-8
 
 Feature: Call5 - Results projection
+
```

Comment:
What about specifying an expression as part of YIELD? I guess that should trigger and error,
`YIELD A+1`

Same idea for yielding none existing outputs

Reply:
sounds good to me